reactioncommerce/reaction-component-library
# feat: add StripePaymentCheckoutAction component

Resolves #216 
Impact: normal
Type: feature

## Component
Adds a checkout action component to house the stripe payment form. Note: The "Use a different billing address" feature is not currently implemented, it's blocked `SelectableList` being completed.

## Testing
1. Enter a shipping address
2. Click on save and continue
3. Click on the "change" link for shipping address
4. Update shipping address
5. Verify change was saved
6. The "Payment information" step should be active
7. Enter a valid test payment info: CC: 4111 1111 1111 1111, valid expire date, any three numbers for cvc and a valid postal code.
8. Click on save and continue
9. Verify payment information was saved.
